documentation/introduction.md

```diff
@@ -5,7 +5,7 @@ can be found on [github](https://github.com/MinaFoundation/mina-fungible-token),
 [npm package](https://www.npmjs.com/package/mina-fungible-token).
 
 The fungible token standard uses Mina's native support for custom tokens (see
-[MIP-4](https://github.com/MinaProtocol/MIPs/blob/main/MIPS/mip-zkapps.md#token-mechanics)). An
+[MIP-4](https://github.com/MinaProtocol/MIPs/blob/main/MIPS/mip-0004-zkapps.md)). An
```

Comment:
Good catch!

Can you add back the `#token-mechanics`, so the link goes directly to the right section in the MIP?

```suggestion
[MIP-4](https://github.com/MinaProtocol/MIPs/blob/main/MIPS/mip-0004-zkapps.md#token-mechanics)). An
```